model/map-file/pom.xml

```diff
@@ -28,15 +28,29 @@
 
     <artifactId>keycloak-model-map-file</artifactId>
     <name>Keycloak Model Map File</name>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>2.3.2</version>
+                <configuration>
+                    <source>11</source>
+                    <target>11</target>
+                </configuration>
```

Comment:
I'm surprise to see the compiler plugin's version and source/target defined here. Shouldn't this be the same for all Keycloak? Could you please add a comment with maybe the error message you were seeing without this?

Reply:
Good spot. This is partly correct - the compiler plugin remained here due to historical reasons and will be fixed with the new commit.
As for forcing the JDK version, it is set to be 11 (due to Keycloak source requirement inconsistency) similarly to e.g. quarkus module: https://github.com/keycloak/keycloak/blob/51bed81814574d254c07f091936483cf97716fa7/quarkus/pom.xml#L52-L54